rsdoctor/modern-minimal/modern.config.ts

```diff
@@ -0,0 +1,33 @@
+import { appTools, defineConfig } from '@modern-js/app-tools';
+import { RsdoctorWebpackPlugin } from '@rsdoctor/webpack-plugin';
```

Comment:
we can use the Rspack mode of Modern.js in this example

Reply:
(which is faster

Reply:
change to rspack mode, rspack mode faster more.